TAO/TAO_IDL/fe/idl.ypp

```diff
@@ -912,41 +914,47 @@ template_module_inst
 
 interface_def
         : interface
+        {
+          $$ = $1;
```

Comment:
Is this needed?  IIRC Bison does this by default.

Reply:
I thought it did, but I was having trouble with the annotation working and I put these in to make sure it was passing the value.